sql/burnham_derived/test_result_v1/schema.bq.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "name": "submission_timestamp",
+    "type": "TIMESTAMP",
+    "description": "Time at which this test report was submitted"
+  },
+  {
+    "name": "test_run",
+    "type": "STRING",
+    "description": "ID of the test run"
+  },
+  {
+    "name": "test_name",
+    "type": "STRING",
+    "description": "Name of the test run"
+  },
+  {
+    "name": "test_outcome",
+    "type": "STRING",
+    "description": "Outcome of the test run; one of ERROR, FAILED, PASSED, SKIPPED, XFAILED, XPASSED"
+  },
+  {
+    "name": "test_duration_seconds",
+    "type": "FLOAT64",
+    "description": "Duration of the test run"
+  },
+  {
+    "name": "test_report",
+    "type": "STRING",
+    "description": "Markdown-formatted report for error or failed tests"
+  }
+]
```

Comment:
As discussed on https://github.com/mozilla/burnham/issues/39#issuecomment-670516119 we also need a `test_log_url` field that contains a URL to the Airflow job.